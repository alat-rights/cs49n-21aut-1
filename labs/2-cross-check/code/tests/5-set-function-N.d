5-set-function-N.o: 5-set-function-N.c test-helper.h ../rpi.h \
  ../fake-pi.h
