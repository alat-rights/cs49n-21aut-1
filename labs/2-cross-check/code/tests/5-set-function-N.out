TRACE: calling pi code
TRACE: GET32(0x20200000) = 0x327b23c6
TRACE: PUT32(0x20200000) = 0x327b23c0
TRACE: GET32(0x20200000) = 0x327b23c0
TRACE: PUT32(0x20200000) = 0x327b23c1
TRACE: GET32(0x20200000) = 0x327b23c1
TRACE: PUT32(0x20200000) = 0x327b23c2
TRACE: GET32(0x20200000) = 0x327b23c2
TRACE: PUT32(0x20200000) = 0x327b23c3
TRACE: GET32(0x20200000) = 0x327b23c3
TRACE: PUT32(0x20200000) = 0x327b23c4
TRACE: GET32(0x20200000) = 0x327b23c4
TRACE: PUT32(0x20200000) = 0x327b23c5
TRACE: GET32(0x20200000) = 0x327b23c5
TRACE: PUT32(0x20200000) = 0x327b23c6
TRACE: GET32(0x20200000) = 0x327b23c6
TRACE: PUT32(0x20200000) = 0x327b23c7
TRACE: GET32(0x20200000) = 0x327b23c7
TRACE: PUT32(0x20200000) = 0x327b23c7
TRACE: GET32(0x20200000) = 0x327b23c7
TRACE: PUT32(0x20200000) = 0x327b23cf
TRACE: GET32(0x20200000) = 0x327b23cf
TRACE: PUT32(0x20200000) = 0x327b23d7
TRACE: GET32(0x20200000) = 0x327b23d7
TRACE: PUT32(0x20200000) = 0x327b23df
TRACE: GET32(0x20200000) = 0x327b23df
TRACE: PUT32(0x20200000) = 0x327b23e7
TRACE: GET32(0x20200000) = 0x327b23e7
TRACE: PUT32(0x20200000) = 0x327b23ef
TRACE: GET32(0x20200000) = 0x327b23ef
TRACE: PUT32(0x20200000) = 0x327b23f7
TRACE: GET32(0x20200000) = 0x327b23f7
TRACE: PUT32(0x20200000) = 0x327b23ff
TRACE: GET32(0x20200000) = 0x327b23ff
TRACE: PUT32(0x20200000) = 0x327b223f
TRACE: GET32(0x20200000) = 0x327b223f
TRACE: PUT32(0x20200000) = 0x327b227f
TRACE: GET32(0x20200000) = 0x327b227f
TRACE: PUT32(0x20200000) = 0x327b22bf
TRACE: GET32(0x20200000) = 0x327b22bf
TRACE: PUT32(0x20200000) = 0x327b22ff
TRACE: GET32(0x20200000) = 0x327b22ff
TRACE: PUT32(0x20200000) = 0x327b233f
TRACE: GET32(0x20200000) = 0x327b233f
TRACE: PUT32(0x20200000) = 0x327b237f
TRACE: GET32(0x20200000) = 0x327b237f
TRACE: PUT32(0x20200000) = 0x327b23bf
TRACE: GET32(0x20200000) = 0x327b23bf
TRACE: PUT32(0x20200000) = 0x327b23ff
TRACE: GET32(0x20200000) = 0x327b23ff
TRACE: PUT32(0x20200000) = 0x327b21ff
TRACE: GET32(0x20200000) = 0x327b21ff
TRACE: PUT32(0x20200000) = 0x327b23ff
TRACE: GET32(0x20200000) = 0x327b23ff
TRACE: PUT32(0x20200000) = 0x327b25ff
TRACE: GET32(0x20200000) = 0x327b25ff
TRACE: PUT32(0x20200000) = 0x327b27ff
TRACE: GET32(0x20200000) = 0x327b27ff
TRACE: PUT32(0x20200000) = 0x327b29ff
TRACE: GET32(0x20200000) = 0x327b29ff
TRACE: PUT32(0x20200000) = 0x327b2bff
TRACE: GET32(0x20200000) = 0x327b2bff
TRACE: PUT32(0x20200000) = 0x327b2dff
TRACE: GET32(0x20200000) = 0x327b2dff
TRACE: PUT32(0x20200000) = 0x327b2fff
TRACE: GET32(0x20200000) = 0x327b2fff
TRACE: PUT32(0x20200000) = 0x327b0fff
TRACE: GET32(0x20200000) = 0x327b0fff
TRACE: PUT32(0x20200000) = 0x327b1fff
TRACE: GET32(0x20200000) = 0x327b1fff
TRACE: PUT32(0x20200000) = 0x327b2fff
TRACE: GET32(0x20200000) = 0x327b2fff
TRACE: PUT32(0x20200000) = 0x327b3fff
TRACE: GET32(0x20200000) = 0x327b3fff
TRACE: PUT32(0x20200000) = 0x327b4fff
TRACE: GET32(0x20200000) = 0x327b4fff
TRACE: PUT32(0x20200000) = 0x327b5fff
TRACE: GET32(0x20200000) = 0x327b5fff
TRACE: PUT32(0x20200000) = 0x327b6fff
TRACE: GET32(0x20200000) = 0x327b6fff
TRACE: PUT32(0x20200000) = 0x327b7fff
TRACE: GET32(0x20200000) = 0x327b7fff
TRACE: PUT32(0x20200000) = 0x32787fff
TRACE: GET32(0x20200000) = 0x32787fff
TRACE: PUT32(0x20200000) = 0x3278ffff
TRACE: GET32(0x20200000) = 0x3278ffff
TRACE: PUT32(0x20200000) = 0x32797fff
TRACE: GET32(0x20200000) = 0x32797fff
TRACE: PUT32(0x20200000) = 0x3279ffff
TRACE: GET32(0x20200000) = 0x3279ffff
TRACE: PUT32(0x20200000) = 0x327a7fff
TRACE: GET32(0x20200000) = 0x327a7fff
TRACE: PUT32(0x20200000) = 0x327affff
TRACE: GET32(0x20200000) = 0x327affff
TRACE: PUT32(0x20200000) = 0x327b7fff
TRACE: GET32(0x20200000) = 0x327b7fff
TRACE: PUT32(0x20200000) = 0x327bffff
TRACE: GET32(0x20200000) = 0x327bffff
TRACE: PUT32(0x20200000) = 0x3263ffff
TRACE: GET32(0x20200000) = 0x3263ffff
TRACE: PUT32(0x20200000) = 0x3267ffff
TRACE: GET32(0x20200000) = 0x3267ffff
TRACE: PUT32(0x20200000) = 0x326bffff
TRACE: GET32(0x20200000) = 0x326bffff
TRACE: PUT32(0x20200000) = 0x326fffff
TRACE: GET32(0x20200000) = 0x326fffff
TRACE: PUT32(0x20200000) = 0x3273ffff
TRACE: GET32(0x20200000) = 0x3273ffff
TRACE: PUT32(0x20200000) = 0x3277ffff
TRACE: GET32(0x20200000) = 0x3277ffff
TRACE: PUT32(0x20200000) = 0x327bffff
TRACE: GET32(0x20200000) = 0x327bffff
TRACE: PUT32(0x20200000) = 0x327fffff
TRACE: GET32(0x20200000) = 0x327fffff
TRACE: PUT32(0x20200000) = 0x321fffff
TRACE: GET32(0x20200000) = 0x321fffff
TRACE: PUT32(0x20200000) = 0x323fffff
TRACE: GET32(0x20200000) = 0x323fffff
TRACE: PUT32(0x20200000) = 0x325fffff
TRACE: GET32(0x20200000) = 0x325fffff
TRACE: PUT32(0x20200000) = 0x327fffff
TRACE: GET32(0x20200000) = 0x327fffff
TRACE: PUT32(0x20200000) = 0x329fffff
TRACE: GET32(0x20200000) = 0x329fffff
TRACE: PUT32(0x20200000) = 0x32bfffff
TRACE: GET32(0x20200000) = 0x32bfffff
TRACE: PUT32(0x20200000) = 0x32dfffff
TRACE: GET32(0x20200000) = 0x32dfffff
TRACE: PUT32(0x20200000) = 0x32ffffff
TRACE: exiting after 129 trace statements
TRACE: pi exited cleanly: 126 calls to random
